api/v1beta1/install/install.go

```diff
@@ -17,4 +17,5 @@ import (
 func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(chaosv1beta1.AddToScheme(scheme))
 	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion))
+	utilruntime.Must(chaosv1beta1.ClientSchemeBuilder.AddToScheme(scheme))
```

Comment:
Have you pull the latest change from the main? It was already fixed by this [PR](https://github.com/DataDog/chaos-controller/pull/857).